lib/MusicBrainz/Server/Edit/ReleaseGroup/SetCoverArt.pm

```diff
@@ -92,12 +92,22 @@ sub accept {
 
 sub foreign_keys {
     my ($self) = @_;
+
+    my $data = $self->data;
+    my $old_release_id = $data->{old}{release_id};
+    my $new_release_id = $data->{new}{release_id};
+
+    my $release_fks = {};
+    if (defined $old_release_id) {
+        $release_fks->{$old_release_id} = ['ArtistCredit'];
+    }
+    if (defined $new_release_id) {
+        $release_fks->{$new_release_id} = ['ArtistCredit'];
+    }
```

Comment:
Same question than reo but about undef old/new release IDs. It would be helpful to explain that in the message of commit <https://github.com/metabrainz/musicbrainz-server/pull/2378/commits/76550239e21642ae2af744a5ee1bf23f4e8d4e25>.

Reply:
The old release ID is null if the cover art was previously set automatically.  See https://musicbrainz.org/edit/86465803 for example.  It doesn't appear that the new release ID is ever null; I thought it was possible to "unset" the chosen release and revert back to choosing the artwork automatically, but that doesn't appear to be possible.

Reply:
Same as above :)